### PR TITLE
Fix guardicore redirection links

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,16 +53,16 @@ The Infection Monkey uses the following techniques and exploits to propagate to
   * WMI
   * Log4Shell
   * Zerologon
-  * and more, see our [Documentation hub](https://www.guardicore.com/infectionmonkey/docs/reference/exploiters/) for more information about our RCE exploiters.
+  * and more, see our [Documentation hub](https://techdocs.akamai.com/infection-monkey/docs/exploiters/) for more information about our RCE exploiters.
 
 ## Setup
-Check out the [Setup](https://www.guardicore.com/infectionmonkey/docs/setup/) page in the Wiki or a quick getting [started guide](https://www.guardicore.com/infectionmonkey/docs/usage/getting-started/).
+Check out the [Setup](https://techdocs.akamai.com/infection-monkey/docs/setting-up-infection-monkey/) page in the Wiki or a quick getting [started guide](https://techdocs.akamai.com/infection-monkey/docs/getting-started/).
```
Comment (shreyamalviya) [inline README.md:59]:
```suggestion
Check out the [Setup](https://techdocs.akamai.com/infection-monkey/docs/setting-up-infection-monkey/) page and the [Getting Started](https://techdocs.akamai.com/infection-monkey/docs/getting-started/) guide in our documentation.
```

File: README.md [hunk 2]
```diff
@@ -53,16 +53,16 @@ The Infection Monkey uses the following techniques and exploits to propagate to
   * WMI
   * Log4Shell
   * Zerologon
-  * and more, see our [Documentation hub](https://www.guardicore.com/infectionmonkey/docs/reference/exploiters/) for more information about our RCE exploiters.
+  * and more, see our [Documentation hub](https://techdocs.akamai.com/infection-monkey/docs/exploiters/) for more information about our RCE exploiters.
```
Comment (shreyamalviya) [inline README.md:56]:
```suggestion
  * and more, see our [documentation hub](https://techdocs.akamai.com/infection-monkey/docs/exploiters/) for more information.
```

**Review by mssalvatore — APPROVED** [review]

**Review by shreyamalviya — CHANGES_REQUESTED** [review]
I think we should rewrite/remove most of the stuff in the README but let's leave it for this PR.

File: README.md (hunk 1 above)
Comment (shreyamalviya) [reply README.md:59]: We should update the links in the Wiki too. Or just remove the Wiki completely.

**Review by shreyamalviya — APPROVED** [review]

**Pull request merged**